src/inverseRobot.js

```diff
@@ -24,7 +24,31 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {}; // a new empty object
+
+  const newRobotKeys = Object.values(robot); /* array of keys for the new
+  object(newRobot)
+  */
+
+  for (const key in robot) {
```

Comment:
for (const key in robot) {
    if (newRobot.hasOwnProperty(robot[key])) {
      return null;
    }
    newRobot[robot[key]] = key;
  }